content/en/security_monitoring/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Security Monitoring
+kind: documentation
+---
+
+## Overview
+
+Datadog Security Monitoring unifies the developers, operations, and security teams into one platform. A single dashboard displays devops content, business metrics, and security content. 
+
+{{< img src="security_monitoring/overview_top.png" alt="Datadog Security Monitoring" responsive="true">}}
+
+Datadog detects threats out-of-the-box and can notify your team of security issues by email, Slack, Jira, PagerDuty, or a webhook. 
+
+{{< img src="security_monitoring/takeover_ex.png" alt="Slack Example" responsive="true" style="width:75%;">}}
+
+Datadog detects many different sorts of threats to your application or infrastructure. A few examples include a targeted attack, an IP communicating with your systems which matches a threat intel list, or an insecure configuration. These threats are surfaced in Datadog as Security Signals and can be correlated and triaged in the [Security Explorer][1]. 
+
+{{< img src="security_monitoring/explorer.png" alt="Security Signals Explorer" responsive="true" >}}
+
+Security Signals are generated by Datadog Security Monitoring with [Detection Rules][2]. Detection Rules detect threats across different sources and are available out of the box for immediate use. Any of the provided detection rules can be cloned to change the configuration. A new rule can also be created from scratch.
```

Comment:
can we link https://app.datadoghq.com/security/configuration/rules/new on the words `new rule`?

Reply:
yup, done